src/js/background.js

```diff
@@ -337,7 +336,7 @@ const µBlock = {  // jshint ignore:line
 
 µBlock.filteringContext = new µBlock.FilteringContext();
 
-globals.µBlock = µBlock;
+self.µBlock = µBlock;
```

Comment:
I believe `µBlock` still needs to be available globally for various reasons. I'm using `self` here, just like in other places.